### DISCUSSION TITLE
obsidian use minio s3 to sync

### DISCUSSION THREAD

**Discussion by JCJC01** [minio/minio#15997] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/103412730/199657963-7a511161-4755-4c81-b54a-adce23e282d3.png)
i want use minio S3 to sync my obsidian data, but i don't know how to fill them
would some big brother kindly show me ?
thanks a lot

**Comment by JCJC01** [comment]
if web minio need set something?

**Comment by ravindk89** [comment, marked as answer]
For endpoint put the full URL and port of your MinIO service.

For Region set it to `us-east-1`. This is the default, unless you override it when you start MinIO

access and secret need to correspond to some user on your MinIO deployment

Bucket name probably just has to exist

For S3 URL Style use Path-style for MinIO.

That's about it.

**Reply by ravindk89** [reply]
https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-single-node-single-drive.html <- this would be sufficient for testing. You'll need to figure out your networking, hostname, etc. That's entirely up to you.
